editor/plugins/tiles/tile_map_editor.cpp

```diff
@@ -55,7 +55,10 @@ void TileMapEditorTilesPlugin::tile_set_changed() {
 }
 
 void TileMapEditorTilesPlugin::_on_random_tile_checkbox_toggled(bool p_pressed) {
-	scatter_spinbox->set_editable(p_pressed);
+	if (p_pressed)
+		scatter_controls_container->show();
+	else
+		scatter_controls_container->hide();
```

Comment:
Our code style requires the use of braces even around single line statements.
You could also do `scatter_controls_container->set_visible(p_pressed);` probably.

Please fix the style issue with `git commit --amend` so that the change stays as a single commit (see https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#modifying-a-pull-request).

Reply:
Okay, I'll try that.

Reply:
Okay, I should have successfully amended the commit. Tested the changes and they seem to work the same.